lib/prometheus_exporter/server/active_record_collector.rb

```diff
@@ -47,11 +47,25 @@ def collect(obj)
       obj["created_at"] = now
 
       @active_record_metrics.delete_if do |current|
-        (obj["pid"] == current["pid"] && obj["hostname"] == current["hostname"]) ||
-          (current["created_at"] + MAX_ACTIVERECORD_METRIC_AGE < now)
+        obj_compare(obj,current, [:pool_name]) || (current["created_at"] + MAX_ACTIVERECORD_METRIC_AGE < now)
```

Comment:
if all we need is to add a check for pool_name do we need to iterate through all the keys and add the general purpose function? 

My pref here is to avoid over generalizing here until we need to.

Reply:
No problem. I didn't know if other's setups would make this entry unique with another key or not as I am somewhat focused atm.